magefile.go

```diff
@@ -152,7 +152,35 @@ func UpdateLibs() error {
 
 // E2e runs e2e tests with a built plugin. Requires docker-compose.
 func E2e() error {
-	return sh.RunV("docker-compose", "--file", "e2e/docker-compose.yml", "up", "--abort-on-container-exit")
+	// The SHA of the envoy version comes from https://github.com/istio/proxy/blob/master/WORKSPACE#L42
+	var envoyImages = []string{
```

Comment:
Let's not change this file and set the env variable in the github actions, I don't think local tasks generally run through multiple versions (user would be able to set the env variable themselves if debugging a CI failure)

Reply:
Agree.